static/app/views/replays/detail/layout/index.tsx

```diff
@@ -123,12 +129,6 @@ function ReplayLayout({layout = LayoutKey.topbar}: Props) {
   }
 
   // layout === 'topbar'
-  const crumbsWithTitle = (
-    <ErrorBoundary mini>
-      <Breadcrumbs showTitle />
-    </ErrorBoundary>
-  );
```

Comment:
This is the version without Tags

Now we're using `sidebarArea` which has tags. See screen shots.